sources/packages/backend/apps/api/src/auth/guards/authorized-parties.guard.ts

```diff
@@ -66,6 +66,9 @@ export class AuthorizedPartiesGuard implements CanActivate {
     authorizedParty: AuthorizedParties,
     identityProvider: IdentityProviders,
   ): boolean {
+    if (authorizedParty === AuthorizedParties.external) {
+      return true;
+    }
```

Comment:
Having this condition here is fine, but we can also have it inside the switch statement, unless I'm missing the consideration not to do so:
```js
switch(authorizedParty) {
    ...
    case AuthorizedParties.external:
        return true;
    default:
        return false;
}
```